docs/phoniebox.conf.example

```diff
@@ -0,0 +1,17 @@
+[SYSTEM]
+BOX_NAME: PhonieBox
+
+[PLAYER]
+AUDIOFOLDERSPATH:"/home/pi/RPi-Jukebox-RFID/shared/"
+PLAYLISTSFOLDERPATH:"/home/pi/RPi-Jukebox-RFID/playlists"
+MUSIC_PLAYER_STATUS: /home/pi/RPi-Jukebox-RFID/shared/music_player_status.json
```

Comment:
To quote, or not to quote, this is the question 😄

Reply:
sure, the quoted one has no yet been used, but lets look forward to yaml...